### PR TITLE
[RFC] nvim.1: Misc. improvements (2)

### PR DESCRIPTION
**Pull request by ghost**
commit message:

```
Besides some minor rewording, this commit does the following:

- the argument to the `q' flag is optional, so denote it as such
- the cross references to vim (`.Xr vim 1') were pointless, just refer
  to it as `Vim'
- the `d' flag doesn't actually take any arguments; `nvim -d' works
  just fine, so make the manual reflect that
- the `L' flag is just an alias for the `r' flag, so clarify that
- there was a lot of redundancy between the `o', `O', and `p' flags,
  unify their descriptions a bit
```

### CODE REVIEW

File: man/nvim.1 [hunk 1]
```diff
@@ -122,7 +118,7 @@ Writing to a file is disabled, but buffers can still be modified.
 .It Fl M
 Disable file and buffer modifications.
 Unsets the options 'write' and 'modifiable'.
-Note that these options can be set to re-enable making modifications.
+Note that these options can be manually set to reallow modifications.
```
Comment (justinmk) [inline man/nvim.1]: just "allow" might work better.

Comment (justinmk) [inline man/nvim.1]: Or leave it as it was, not sure this reads any better.

File: man/nvim.1 [hunk 2]
```diff
@@ -140,7 +136,7 @@ Sets the options 'fkmap' and 'rightleft'.
 Hebrew mode.
 Sets the options 'hkmap' and 'rightleft'.
 .It Fl V Ns Oo Ar N Oc Ns Op Ar file
-Verbose.
+Be verbose.
```
Comment (justinmk) [inline man/nvim.1]: suggest "Verbose mode", it has parallel form with "Debugging mode" and "Recovery mode".

File: man/nvim.1 [hunk 3]
```diff
@@ -160,18 +156,23 @@ Disable the use of swap files.
...
-.It Fl r , L
-List swap files with information about recovering them.
-.It Fl r Ar file
+.It Fl r Op Ar file
 Recovery mode.
-The swap file
+If
+.Ar file
+is omitted
+then list swap files with information about recovering them.
```
Comment (justinmk) [inline man/nvim.1]: "information about recovering them" => "recovery instructions" ?

File: man/nvim.1 [hunk 4]
```diff
@@ -219,29 +220,13 @@ is less than the number of file arguments, allocate windows for the first
...
-.Ar N
-is omitted, open one window for each file.
-If
-.Ar N
-is less than the number of file arguments, allocate windows for the first
-.Ar N
-files and hide the rest.
+Like
+.Fl o ,
+but stack windows vertically.
```
Comment (justinmk) [inline man/nvim.1]: "stack" => "tile" ? Otherwise good reduction.

File: man/nvim.1 (hunk 1 above)
Comment (Pyrohh) [inline man/nvim.1]: Sure, I don't think it's clear whether this is better.

File: man/nvim.1 (hunk 2 above)
Comment (Pyrohh) [inline man/nvim.1]: :+1:

File: man/nvim.1 (hunk 3 above)
Comment (Pyrohh) [inline man/nvim.1]: I like "recovery instructions", good idea.

File: man/nvim.1 (hunk 4 above)
Comment (Pyrohh) [inline man/nvim.1]: sure

**Comment by Pyrohh** [comment]
Thanks for the quick feedback @justinmk, repushed.

**Comment by justinmk** [comment]
LGTM

**Comment by Pyrohh** [comment]
@fmoralesc would you like to take a peek, or shall I just merge it?

**Comment by justinmk** [comment]
I say go ahead, I believe he's going to be away for awhile.

**Comment by Pyrohh** [comment]
Mkay, done.

**Comment by fmoralesc** [comment]
RIP me indeed.

**Pull request merged**